normandy/recipes/models.py

```diff
@@ -45,6 +45,17 @@ def __repr__(self):
         return "<Channel {}>".format(self.slug)
 
 
+class WindowsVersion(models.Model):
+    slug = models.CharField(max_length=255, unique=True)
```

Comment:
To quote [the Django docs](https://docs.djangoproject.com/en/3.0/ref/models/fields/#slugfield):

> Slug is a newspaper term. A slug is a short label for something, containing only letters, numbers, underscores or hyphens. They’re generally used in URLs.

I don't think this field should be named slug. Perhaps `nt_version`, and make it a numeric type instead?

Reply:
ok